### ISSUE TITLE
Perl error after RHEL upgrade

### ISSUE THREAD

**Issue by ubzmrh** [netdisco/netdisco#1180]
My server has just been upgraded from RHEL 7.9 to RHEL 8.9.
I have successfully restart PostgreSQL 
But, when I restart Netdisco, I get this error :
perl: symbol lookup error: /appli/netdisco/perl5/lib/perl5/x86_64-linux-thread-multi/auto/List/Util/Util.so: undefined symbol: Perl_xs_apiversion_bootcheck

I tried upgrade Netdisco with the last release: same error 

The installed perl version is : 
```
perl -v

This is perl 5, version 26, subversion 3 (v5.26.3) built for x86_64-linux-thread-multi
(with 58 registered patches, see perl -V for more detail)

Copyright 1987-2018, Larry Wall

Perl may be copied only under the terms of either the Artistic License or the
GNU General Public License, which may be found in the Perl 5 source kit.

Complete documentation for Perl, including FAQ lists, should be found on
this system using "man perl" or "perldoc perl".  If you have access to the
Internet, point your browser at http://www.perl.org/, the Perl Home Page.
```

**Comment by inphobia** [comment]
sounds very much like
https://github.com/netdisco/netdisco/wiki/Troubleshooting#after-os-update-or-upgrade-netdisco-fails

**Comment by ubzmrh** [comment]
Ok !
After deleting ~/perl5 and a new install Netdisco could start. 
But, new problem : when I start netdisco-web, I can't connect on web interface ! 
Why ?
```
2024/03/07-11:04:42 Starman::Server (type Net::Server::PreFork) starting! pid(475077)
Resolved [*]:5000 to [::]:5000, IPv6
Not including resolved host [0.0.0.0] IPv4 because it will be handled by [::] IPv6
Binding to TCP port 5000 on host :: with IPv6
2024/03/07-11:04:42 Can't connect to TCP port 5000 on :: [Invalid argument]
  at line 66 in file /appli/netdisco/perl5/lib/perl5/Net/Server/Proto/TCP.pm
2024/03/07-11:04:42 Received QUIT. Running a graceful shutdown
2024/03/07-11:04:42 Worker processes cleaned up
2024/03/07-11:04:42 Server closing!
```

For netdisco-web to allow connections, I had to add this instruction to netdisco-web.service (We use SystemD): 
`Environment=IPV=4`
